src/main/webapp/i18n/en/quizExercise.json

```
@@ -267,10 +268,16 @@
                 "individual": "Individual",
                 "explanation": "Batch Mode controls how students can participate in the quiz.\n\nSynchronized: There is exactly opportunity for all students to participate in the quiz at once.\nBatched: There are multiple opportunities for students to participate in the quiz by using a password.\nIndividual: Students can participate in the quiz by themselves at any time while it is released."
             },
+            "releaseDateExplanation": {
+                "synchronized": "Synchronized: The date when the exercise becomes visible to students. Students cannot see the questions until the start date.",
+                "batched": "Batched: The date when the exercise becomes visible to students. Students cannot see the questions until a batch is started and they enter the password for it.",
+                "individual": "Individual: The date when the exercise becomes visible to students. Students can see the questions at the time they choose."
```

```suggestion
                "individual": "Individual: The date when the quiz becomes visible to students. Students can start working on the quiz from this time onward"
```
They cannot only see the quiz in the exercise overview, but can also start working on the quiz right away?

---

Yes, that is also true.